README.md

```diff
@@ -6,3 +6,145 @@
 This repository contains mock server that is used for advanced E2E tests for our proxy integrations.
 When the integration is properly configured, it allows us to inspect requests sent from these integrations and perform necessary assertions.
 
+
+## Preperation
+
+In order to test your proxy integration, you need to configure it to send requests to this mock app.
+In most cases, this can be done by setting these two ENV variables when building the integration: `FPCDN` and `INGRESS_API`. For example:
+
+```bash
+FPCDN=<API_URL> INGRESS_API=<API_URL> pnpm build
+```
+
+Refer to specific integration README for more details.
+
+> **Note**
+> In order to get the API URL, contact the DX team.
+
+## How to use it
+
+This app exposes REST API that can be used to run the tests.
+
+To run tests, send `POST` request to: `/api/test/run-tests` with following payload:
+```json
+{
+  // Your proxy host
+  "host": "https://mock-test-inter-568-mock-app-tests.cfi-fingerprint.com",
```

Comment:
I've asked this to @Orkuncakilkaya during UI review as well: is it possible to get rid of `host` field, and just provide full URL in `cdn` and `ingress` fields instead of paths? It will mean one less field for the user @TheUnderScorer

Reply:
That depends. To me it doesn't seem to be more convenient:
- As user, you need to copy-paste the host name twice with the cdn and ingress paths
- In the app, we need to decide from which parameter (`cdn` or `ingress`) we want to extract hostname for the tests.

IMO, I would prefer to leave it as it is.

Reply:
"As user, you need to copy-paste the host name twice with the cdn and ingress paths"
Assuming that the user is configuring the Pro Agent, usually with a subdomain, they already have the full URL. In my case, I had to manually split the host and the path each time. This is just my feedback, let's see what others think.

"In the app, we need to decide from which parameter (cdn or ingress) we want to extract hostname for the tests."
Why do you need to extract hostname from cdn/ingress request URLs?

Reply:
I've decided to change it in the end, I like the idea of having one parameter less. The change is in this PR: https://github.com/fingerprintjs/dx-team-mock-for-proxy-integrations-e2e-tests/pull/18